javers-core/src/main/java/org/javers/core/graph/LiveGraphFactory.java

```diff
@@ -44,6 +45,10 @@ private Object wrapTopLevelContainer(Object handle){
         if (handle instanceof  Set){
             return new SetWrapper((Set)handle);
         }
+        if (handle.getClass().isArray()){
+            //return new ArrayWrapper(convertToObjectArray(handle));
+            return new ArrayWrapper((Object[])handle);
```

Comment:
@dmmiller612, I've removed this conversion and the tests are still passing, not sure what is it for ...
If this version is ok for you, I can merge